### PR TITLE
chore: demonstrate using self hosted Ollama & OpenAI as BaseModel

### PR DESCRIPTION
**Pull request by mkhludnev**
#2280
Note: it requires LiteLLM [v1.28.9](https://github.com/BerriAI/litellm/releases/tag/v1.28.9) since it needs https://github.com/BerriAI/litellm/pull/2294 https://github.com/BerriAI/litellm/pull/2281, otherwise one can subclass LiteLLMModel, override `_init_environment()` and just refuse to call `validate_environment` since it's broken. I can provide a snippet if anyone is interested.

### CODE REVIEW

**Comment by mkhludnev** [comment]
I have read the CLA Document and I hereby sign the CLA

**Comment by mkhludnev** [comment]
recheck

File: packages/phoenix-evals/src/phoenix/evals/models/litellm.py [hunk 1]
```diff
@@ -65,7 +65,7 @@ def _init_environment(self) -> None:
             from litellm import validate_environment
 
             self._litellm = litellm
-            env_info = validate_environment(self._litellm.utils.get_llm_provider(self.model))
```
Comment (anticorrelator) [inline packages/phoenix-evals/src/phoenix/evals/models/litellm.py:68]: is this no longer needed?

**Review by anticorrelator — COMMENTED** [review]
This looks great! One last ask: could both tests be moved under `tests/phoenix/evals/models/test_litellm.py`?

File: packages/phoenix-evals/src/phoenix/evals/models/litellm.py (hunk 1 above)
Comment (mkhludnev) [reply packages/phoenix-evals/src/phoenix/evals/models/litellm.py:68]: it looks like 
https://github.com/BerriAI/litellm/blob/6c79bea17791bf5781f6d656f641eadcc41dcf39/litellm/utils.py#L5588
however, it means, we need to bump `litellm` up to [v1.28.9](https://github.com/BerriAI/litellm/releases/tag/v1.28.9) in pyproject.toml.

Comment (anticorrelator) [reply packages/phoenix-evals/src/phoenix/evals/models/litellm.py:68]: Let's revert this change if it's not strictly necessary? I don't think we need to bump the version for this

**Comment by mikeldking** [comment]
@mkhludnev thanks so much for your contribution!

**Review by anticorrelator — APPROVED** [review]

File: packages/phoenix-evals/src/phoenix/evals/models/litellm.py (hunk 1 above)
Comment (anticorrelator) [reply packages/phoenix-evals/src/phoenix/evals/models/litellm.py:68]: Actually I think it's probably a good idea to bump the version, I can do that on my end, thanks so much for the PR @mkhludnev!

**Pull request merged**